formatTest/typeCheckedTests/expected_output/arityConversion.re

```diff
@@ -16,11 +16,11 @@ module Test = {
     | Or (int, int);
 };
 
-Test.And (1, 2);
+let _ = Test.And (1, 2);
```

Comment:
Does this fix the following issue? https://github.com/facebook/reason/issues/843

Also, would the output change depending on if we're downconverting to 4.02.3 or not? I recall there was a change in how top level Evals were represented in the AST.

Is there a way to distinguish `let _ = expression;` and `expression;` in the top level of a module?